compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -134,37 +134,16 @@ object Completion {
    * If several denotations share the same name, the type denotations appear before term denotations inside
    * the same `Completion`.
```

Comment:
It seems this part of the comment should be deleted

Reply:
I updated the comment instead for clarity